### ISSUE TITLE
1.0.6 Linux client lacks testing.trset: Error: ENOENT: no such file or directory, open './testing.trset'

### ISSUE THREAD

**Issue by tomreyn** [ixmaps/IXmapsClient#111]
The [1.0.6 Linux client](https://www.ixmaps.ca/IXmapsClient/IXmapsClient.1.0.6.linux.tar.gz) lacks testing.trset. As a result, you run into this when selecting TR set "testing" on the web UI and confirming the choice by clicking on "Generate Traceroutes":

```
Error: ENOENT: no such file or directory, open './testing.trset'
    at Error (native)
    at Object.fs.openSync (fs.js:549:18)
    at Object.fs.readFileSync (fs.js:397:15)
    at Object.exports.getTrset (/home/x/IXmapsClient/lib/trset.js:34:33)
    at Object.exports.submitTraceOptions (/home/x/IXmapsClient/lib/processor.js:31:12)
    at Socket.submitTrace (/home/x/IXmapsClient/server.js:149:17)
    at emitOne (events.js:77:13)
    at Socket.emit (events.js:169:7)
    at Socket.onevent (/home/x/IXmapsClient/node_modules/socket.io/lib/socket.js:348:8)
    at Socket.onpacket (/home/x/IXmapsClient/node_modules/socket.io/lib/socket.js:308:12)
```

**Comment by Andrew-Clement** [comment]
Thanks Tom, for bringing this to our attention.

We’ll look into this issue.

Andrew.


On Dec 31, 2019, at 10:42 PM, Tom Reynolds <notifications@github.com> wrote:

The 1.0.6 Linux client <https://www.ixmaps.ca/IXmapsClient/IXmapsClient.1.0.6.linux.tar.gz> lacks testing.trset. As a result, you run into this when selecting TR set "testing" on the web UI and confirming the choice by clicking on "Generate Traceroutes":

Error: ENOENT: no such file or directory, open './testing.trset'
    at Error (native)
    at Object.fs.openSync (fs.js:549:18)
    at Object.fs.readFileSync (fs.js:397:15)
    at Object.exports.getTrset (/home/x/IXmapsClient/lib/trset.js:34:33)
    at Object.exports.submitTraceOptions (/home/x/IXmapsClient/lib/processor.js:31:12)
    at Socket.submitTrace (/home/x/IXmapsClient/server.js:149:17)
    at emitOne (events.js:77:13)
    at Socket.emit (events.js:169:7)
    at Socket.onevent (/home/x/IXmapsClient/node_modules/socket.io/lib/socket.js:348:8)
    at Socket.onpacket (/home/x/IXmapsClient/node_modules/socket.io/lib/socket.js:308:12)

**Comment by colinmccann** [comment]
Resolved with 1.1.1 release
